pkg/controller/constraint/constraint_controller.go

```diff
@@ -47,72 +46,81 @@ var (
 )
 
 const (
-	finalizerName = "finalizers.gatekeeper.sh/constraint"
+	finalizerName      = "finalizers.gatekeeper.sh/constraint"
+	constraintsGroup   = "constraints.gatekeeper.sh"
+	constraintsVersion = "v1beta1"
```

Comment:
the version can change so let's not hardcode this

Reply:
Good callout 👍